packages/varlet-cli/src/shared/ejsRender.ts

```diff
@@ -0,0 +1,52 @@
+import ejs from 'ejs'
+import fs from 'fs-extra'
+import path from 'node:path'
+import prettier from 'prettier'
+
+export async function ejsRender(filePath: string, options: any): Promise<void> {
+  try {
+    let prettierCode = ''
+    // 根目录template 绝对路径
+    const templatePath = path.resolve(__dirname, `../../template/create`)
+    // 获取当前渲染文件的 各种 参数 such as ext
+    const file = path.parse(filePath)
+    // 编译根目录 创建的根目录
+    const dest = path.resolve(`${process.cwd()}/src`, options.projectName)
+    // 当前 需要编译的 ejs文件
+    const readFilePath = path.resolve(dest, file.dir, `${file.name}.ejs`)
+    // 转换 之后的 js or ts or vue 文件
+    const outputFilePath = path.resolve(dest, filePath)
+    // 是一个buffer
+    const templateCode = await fs.readFile(readFilePath)
+    // 编译当前code
+    const code = ejs.render(templateCode.toString(), options)
```

Comment:
you can `const templateCode = await fs.readFile(readFilePath, { encoding: 'utf-8' })`, then remove `toString`